_posts/2021-09-01-logical-operators-short-circuiting.md

```diff
@@ -65,31 +65,23 @@ print(not (not (false and 15)))
 ```
 
 ### Example usage of NOT
-Let’s say you have a `GuiObject` which you want to toggle its `Visible` property value on and off each time it's clicked. A naive solution would be:
-
+Let’s say you have a `guiObject` which you want to toggle its `Visible` property’s value on and off each time a `guiButton` is clicked. A naive solution would be:
 ```lua
--- Assuming guiObject is defined properly, this would work.
-guiObject.MouseButton1Click:Connect(function()
-    -- Parenthesis around the condition isn’t necessary.
+guiButton.MouseButton1Click:Connect(function()
     if guiObject.Visible then
         guiObject.Visible = false
     else
         guiObject.Visible = true
     end
 end)
 ```
-
```

Comment:
These spacings are important due to how markdown formatting works, please redo

Reply:
> These spacings are important due to how markdown formatting works, please redo

Alright